src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -132,11 +132,17 @@ function getListBlockType(
 function getBlockMapSupportedTags(
   blockRenderMap: DraftBlockRenderMap
 ): Array<string> {
+  // Some blocks must be treated as unstyled when not present on the blockRenderMap
   const unstyledElement = blockRenderMap.get('unstyled').element;
-  return blockRenderMap
+  const defaultUnstyledSet = new Immutable.Set(['p']);
```

Comment:
Hi @mitermayer the PR is here: https://github.com/facebook/draft-js/pull/584 -- everything works correctly when I play around with it locally. If you know who I can talk to to get the PR reviewed, that would be great. Thanks!